packages/types/src/documents/app/query.ts

```diff
@@ -8,6 +8,8 @@ export interface QuerySchema {
   subtype?: string
 }
 
+export type QueryVerb = "read" | "create" | "update" | "delete"
```

Comment:
Are these query verbs not used for mongo queries, as well?

Reply:
That's a good point, MongoDB appears to also support `"aggregate"`. If I add that here, though, I get some type check failures because `aggregate` is not a function on `IntegrationBase`.

I think the reality is that typing around this area isn't great today, and it's probably a bigger piece of work to fix it in full. I've improved it a tiny bit in this PR but I think I'd like to just leave it at that for now.